tests/test_core/test_alignment.py

```diff
@@ -244,6 +244,16 @@ def setUp(self):
         self.b = Alignment(["AAA", "AAA"])
         self.c = SequenceCollection(["AAA", "AAA"])
 
+    def test_deepcopy(self):
+        """correctly deep copy aligned objects in an alignment"""
+        path = "data/brca1_5.paml"
+        aln = load_aligned_seqs(path, array_align=False, moltype="dna")
```

Comment:
create a small dictionary of sequence data (look at one of the other test methods) and just pass that directly to self.Class, along with specifying the moltype. A dict with 2 sequences is enough, I think.